content/F2015/F15-event-dropbox-coffee-chats.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug: dropbox-coffee-chats-f15
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+Note: This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
If it's something we're going to adopt site-wide, an issue should be opened for it so that it can all be done at once instead of one page at a time.